src/components/DocFooter.tsx

```diff
@@ -6,20 +6,25 @@ import { useDocContext } from '~/src/hooks/useDocContext';
 
 import { capitalize } from '../lib/str';
 
-function parseLink(link?: string) {
-  if (!link) return;
+function parseLink(link: string, path: string) {
   if (link.startsWith('../')) {
     link = link.replace('../', '');
   }
-  link = `/docs/${link}`;
+  link = path.startsWith('/guides/')
+    ? link.replace('guides/', '')
+    : `/docs/${link}`;
   return link;
 }
 
 export function DocFooter() {
   const { docLink, doc } = useDocContext();
   const router = useRouter();
-  const prevLink = parseLink(docLink.prev?.slug);
-  let nextLink = parseLink(docLink.next?.slug);
+  const prevLink = docLink.prev?.slug
+    ? parseLink(docLink.prev.slug, router.asPath)
+    : undefined;
+  let nextLink = docLink.next?.slug
+    ? parseLink(docLink.next?.slug, router.asPath)
+    : undefined;
```

Comment:
You can omit `undefined` by using a nullish coalescing
```suggestion
  const prevLink = docLink.prev?.slug ?? parseLink(docLink.prev.slug, router.asPath)
  let nextLink = docLink.next?.slug ?? parseLink(docLink.next?.slug, router.asPath)
```